typing/typetexp.mli

```diff
@@ -28,7 +28,6 @@ val transl_simple_type_delayed:
 val transl_type_scheme:
         Env.t -> Parsetree.core_type -> Typedtree.core_type
 val reset_type_variables: unit -> unit
-val type_variable: Location.t -> string -> type_expr
```

Comment:
Why are you removing this function? It looks like it could be convenient for compiler-libs user. Is it doing the wrong thing? (If I understand the patch correctly, the `Unbound_type_variable` error that is caught in `report` is raised by a different site?)